test/tests/ver-1d/ver-1d-trapping.i

```diff
@@ -144,13 +165,19 @@ temperature = 1000
 [Executioner]
   type = Transient
   end_time = 1000
-  dt = 1
+  dtmax = 5
   solve_type = NEWTON
+  scheme = BDF2
+  petsc_options_iname = '-pc_type'
+  petsc_options_value = 'lu'
   line_search = 'none'
-  automatic_scaling = true
-  verbose = true
-  compute_scaling_once = false
-  l_tol = 1e-11
+  [TimeStepper]
+    type = IterationAdaptiveDT
+    dt = 1e-6
```

Comment:
Do we need such a small initial time step?

Reply:
Yes, this is one of the things that prevents the oscillations. We might get away with slightly larger intial dt, but I found it makes the simulation very sensitive to the material properties and model parameters. In general it is also a good idea to start with smaller than necessary timestep sizes to minimize numerical error.